docs/getting_started.md

```diff
@@ -287,7 +287,7 @@ We add the component to our pipeline definition and specify that it depends on t
 We can now easily run our new pipeline:
 
 ```bash
-fondant run pipeline:my_pipeline --local
+fondant run pipeline --local
```

Comment:
I think one command is outdated here. Either the one in line 142 `fondant run pipeline.py --local` or this one.